Documentation/Tutorials/edge-send-event-tutorial.md

```diff
@@ -0,0 +1,749 @@
+
+1. Make sure that the mobile property used has Assurance installed, and that the property has been properly published.
+2. Make sure that the mobile property ID is set in the `ENVIRONMENT_FILE_ID` variable in `AppDelegate.swift`, then rebuild the app.
+
+</p></details>
+
+
+<details>
+  <summary> Connecting using QR code </summary><p>
+
+To connect using QR code:
+Prerequisites (see [Set up the Assurance session](#1-set-up-the-assurance-session) for details on QR code requirements):
+- Running app on a **physical device** with camera that can scan QR codes
+- App URL for deep linking is configured
+- App code for receiving link and connecting to Assurance is implemented
+
+1. Use physical device's camera to scan the QR code, which when tapped, should trigger a confirmation dialog to open the app.
+2. App should open and show the Assurance PIN screen to authenticate the session connection; enter the PIN from the session details and tap **Connect**
+
+</p></details>
+
+Once connected to Assurance, in the tutorial app, an Adobe Experience Platform icon (**1**) will appear in the top right corner of the screen with a green dot indicating a connected session.  
+<img src="../Assets/edge-send-event-tutorial/ios-assurance-connection.png" alt="Assurance Session Start - Web UI after connection" width="400"/>  
+
+In the web-based Assurance session, there is also an indicator in the top right that shows the number of connected sessions (which in this case should now show a green dot with "1 Client Connected" (**1**)).  
+<img src="../Assets/edge-send-event-tutorial/assurance-session-start.jpg" alt="Assurance Session Start - Web UI after connection" width="1100"/>  
+
+Notice how in the Assurance session Events view (**2**), there are already events populating as a consequence of the connection of the mobile app to the Assurance session (**3**); the Assurance extension itself emits events about the session connection and subsequently captures these events to display in the web-based session viewer. You can expect Assurance to capture all events processed by the AEP SDK from all other extensions as well.  
+
+### 3. Assurance Event transactions view - check for Edge events  
+In order to see Edge events, in the connected app instance:
+1. Tap either **Product add event** or **Product view event** to send an Experience Event to the Edge Network! 
+   - Behind the scenes the buttons use the `sendEvent` API from the Edge extension. This event will be captured by the Assurance extension and shown in the web session viewer.
+
+<img src="../Assets/edge-send-event-tutorial/ios-trigger-event.png" alt="Simulator tracking buttons" width="400"/>
+
+1. Click the `AEP Request Event` event (**1**) in the events table to see the event details in the right side window.
```

Comment:
might need some section titles before resetting the step numbers to 1, e.g. you could have:
#### Inspect events with Assurance
##### Send product related Experience events
text + steps
##### Inspect events using Events view
...
##### Inspect events using Event Transactions view
...